hazelcast-client/src/main/resources/hazelcast-client-config-3.10.xsd

```diff
@@ -668,7 +668,7 @@
                     <xs:documentation>
                         For how long the pre-fetched IDs can be used. If this time elapses, new IDs will be fetched.
                         Time unit is milliseconds.
-                        If value is &lt;= 0, validity is unlimited. Default value is 10000 (10 seconds).
+                        If value is &lt;= 0, validity is unlimited. Default value is 600,000 (10 minutes).
```

Comment:
The default value mentioned here is not enforced in the XSD; even though it will eventually be applied, I think it's best to have the defaults also visible in the XSD (same goes for the member-side XSD as well, in `hazelcast-spring` the default is already in the schema).